vertical-pod-autoscaler/pkg/updater/priority/update_priority_calculator.go

```diff
@@ -163,6 +165,35 @@ func (calc *UpdatePriorityCalculator) GetSortedPods(admission PodEvictionAdmissi
 	return result
 }
 
+// GetProcessedRecommendationTargets takes a RecommendedPodResources object and returns a formatted string
+// with the recommended pod resources. Specifically, it formats the target and uncapped target CPU and memory.
+func (calc *UpdatePriorityCalculator) GetProcessedRecommendationTargets(r *vpa_types.RecommendedPodResources) string {
+	sb := &strings.Builder{}
+	for _, cr := range r.ContainerRecommendations {
+		sb.WriteString(fmt.Sprintf("%s: ", cr.ContainerName))
+		if cr.Target != nil {
+			sb.WriteString("target: ")
+			if !cr.Target.Memory().IsZero() {
+				sb.WriteString(fmt.Sprintf("%sK ", cr.Target.Memory().AsDec()))
```

Comment:
I think we need to scale this in order to be Kilobytes, otherwise `AsDec()` returns the memory value in Bytes

```suggestion
				sb.WriteString(fmt.Sprintf("%sk ", cr.Target.Memory().ScaledValue(resource.Kilo)))
```

whereas `resource` is imported from 
`"k8s.io/apimachinery/pkg/api/resource"`

Reply:
Good point - added this in the latest commit + updated the tests!